src/Headers.test.ts

```diff
@@ -246,6 +254,13 @@ describe('.append()', () => {
     headers.append('accept', 'image/png')
     expect(headers.get('accept')).toEqual('*/*, image/png')
   })
+
+  it('does not duplicate values with the existing header', () => {
+    const headers = new Headers({ accept: '*/*' })
+
+    headers.append('Accept', '*/*')
```

Comment:
Appending an existing header with a different casing _and the same value_ does not duplicate the header value.